mypy/checker.py

```diff
@@ -1647,6 +1647,24 @@ def visit_class_def(self, defn: ClassDef) -> None:
                 with self.scope.push_class(defn.info):
                     self.accept(defn.defs)
             self.binder = old_binder
+            for base in typ.mro[1:]:
+                if base.name() != 'object' and base.defn.info:
```

Comment:
Checking for `base.defn.info` looks suspicious, at this stage it must be always present, if it isn't, it is a bug elsewhere. Is this actually needed?

Reply:
actually there are AssertionErrors during tests, but I didn't succeed to reproduce them myself..